graphqlapi/src/main/java/io/stargate/graphql/web/ApplicationConfiguration.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.graphql.web;
+
+import io.dropwizard.Configuration;
+
+public class ApplicationConfiguration extends Configuration {}
```

Comment:
If this is empty, maybe we can create an Application starer with the default `Configuration` instead of creating this class? Or maybe add an explanation of why this is empty.

Reply:
This is empty since we use yaml config. It still needs to be here so we can extend it in io.stargate.graphql.web.Server and set up the bootstrap stuff.

Reply:
Yes you're right. I copy-pasted from the REST bundle and didn't put much thought into it, but that class is indeed unnecessary.